### PR TITLE
Change build shield from Travis to CircleCI

### LINKED ISSUE #296: Build shield in README is still linked to Travis CI
## Expected Behavior

Build shield in README should be linked to CircleCI.

## Actual Behavior

Build shield in README is still linked to Travis CI

## Possible solution

Changing [line 12 in README.md](https://github.com/physiopy/phys2bids/blob/master/README.md#L12) to the following:
`[CircleCI](https://circleci.com/gh/physiopy/phys2bids)`

### PR DESCRIPTION
**Pull request by eurunuela**
Closes #296

## Proposed Changes

  - Change link to Travis CI for the link to CircleCI to display correct build shield.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ phys2bids
 [![Join the chat at Gitter: https://gitter.im/physiopy/phys2bids](https://badges.gitter.im/physiopy/phys2bids.svg)](https://gitter.im/physiopy/phys2bids?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)
 [![codecov](https://codecov.io/gh/physiopy/phys2bids/branch/master/graph/badge.svg)](https://codecov.io/gh/physiopy/phys2bids)
 
-[![Build Status: Linux](https://travis-ci.org/physiopy/phys2bids.svg?branch=master)](https://travis-ci.org/physiopy/phys2bids)
+[![CircleCI](https://circleci.com/gh/physiopy/phys2bids.svg?style=shield)](https://circleci.com/gh/physiopy/phys2bids)
```
Comment (tsalo) [inline README.md:12]:
```suggestion
[![CircleCI](https://circleci.com/gh/physiopy/phys2bids.svg?branch=master&style=shield)](https://circleci.com/gh/physiopy/phys2bids)
```

To only show the status on the master branch.

Comment (eurunuela) [reply README.md:12]: I thought that was the default. Thanks!

**Comment by tsalo** [comment]
You'll want to do `docs/index.rst` too.

**Comment by eurunuela** [comment]
> You'll want to do `docs/index.rst` too.

What do you mean? We already have a badge for the docs.

**Comment by tsalo** [comment]
The badge in `docs/index.rst` still points to Travis.

Apologies for the blame view, but permalinks in RST and MD files don't work...

https://github.com/physiopy/phys2bids/blame/ff2e0e36cf6c3a2170550defbf2570114938513a/docs/index.rst#L13-L15

**Comment by eurunuela** [comment]
Oh right. I thought you were talking about the README. My bad. Will push a commit now.

**Comment by eurunuela** [comment]
Okay, I think that should make it @tsalo . Thank you!

**Comment by tsalo** [comment]
@eurunuela the build docs artifact is pointing to the wrong build. Based on https://github.com/larsoner/circleci-artifacts-redirector/issues/6, I think the solution may be to enable **Only build pull requests**.

CircleCI phys2bids page --> **Project Settings** --> **Advanced Settings** --> turn on **Only build pull requests**

**Comment by eurunuela** [comment]
> @eurunuela the build docs artifact is pointing to the wrong build. Based on [larsoner/circleci-artifacts-redirector#6](https://github.com/larsoner/circleci-artifacts-redirector/issues/6), I think the solution may be to enable **Only build pull requests**.
> 
> CircleCI phys2bids page --> **Project Settings** --> **Advanced Settings** --> turn on **Only build pull requests**
> 
> Would you be willing to give that a try?

Done. Please let me know if there's anything we have to change.

**Comment by eurunuela** [comment]
Just swapped the Gitter and CircleCI badges on the docs. It makes more sense to me that all the ones related to builds are next to each other.

I don't have a strong opinion on this, so let me know if you think it's not a good idea.

**Comment by tsalo** [comment]
It turns out I have permissions on the CircleCI settings for this project, so I checked them. After you enabled "Only build pull requests", I noticed that the CircleCI status checks didn't show up here on your newest commit. I went into the settings and enabled "Build forked pull requests", which should hopefully fix that. A bunch of pipelines started running, and I'm hoping that, once they're done, everything should be good here. All of the settings match my own on other projects I contribute to, so hopefully it'll be fine. I just want to wait until it all finishes before approving this.

**Comment by tsalo** [comment]
The backlog finished. Would you mind pushing one small commit to hopefully trigger things a final time?

**Comment by eurunuela** [comment]
Thanks @tsalo . Hopefully this last commit will make it work.

**Comment by eurunuela** [comment]
> It turns out I have permissions on the CircleCI settings for this project, so I checked them. After you enabled "Only build pull requests", I noticed that the CircleCI status checks didn't show up here on your newest commit. I went into the settings and enabled "Build forked pull requests", which should hopefully fix that. A bunch of pipelines started running, and I'm hoping that, once they're done, everything should be good here. All of the settings match my own on other projects I contribute to, so hopefully it'll be fine. I just want to wait until it all finishes before approving this.

It turns out I had to do that on my own CircleCI profile too, otherwise the checks only appear when merging with master.

**Pull request merged**